src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -2054,6 +2054,10 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
 
             var textBlock = values[0] as TextBlock;
             var viewModel = values[1] as SearchViewModel;
+
+            if(viewModel == null)
```

Comment:
Hi @aosyatnik, this is a great explanation, but why don't you embed that alongside the code so that future readers get to understand this check too?